spec/book_spec.rb

```diff
@@ -0,0 +1,15 @@
+require './book'
```

Comment:
- Good Job on implementing tests for all files asked but there are still some methods that didn't have any test Kindly add tests for the following methods as per this [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_unit_tests.md#:~:text=Add%20unit%20tests%20for%20all%20the%20methods%20in%20all%20the%20classes%3A). They asked to add tests for all methods in all classes. The missing methods are:
   - [ ] `add_rental` and `to_h` in the Book class.
   - [ ] `add_rental` and `correct_name` in the Person class.
   - [ ] `add_classroom` and `to_h` in the Student class.
   - [ ] `to_h` in the Teacher class.
   - [ ] `add_student` in the Classroom class.
   - [ ] `to_h` in the Rental class.
   - [ ] `correct_name` in the Nameable class.
   
for some of the previous methods, you test it manually without using the function. Kindly use the function instead.

Reply:
Thanks for the info @gemmen29 . We added required tests to the project.